tests/unit/migrations.py

```diff
@@ -163,6 +164,9 @@ def create_bigint_db():
 
         db.session.commit()
 
+        db.session.query(User).get(1).hashed_password = 'pass'
```

Comment:
Rather than setting the hashed_password field to the cleartext password, can we just hard-code the hashed output (and add a comment re: what the password is)? Would be nice if we kept the database contents actually valid.